tests/e2e-kubernetes/testsuites/credentials.go

```diff
@@ -870,6 +998,36 @@ func overrideServiceAccountRole(ctx context.Context, f *framework.Framework, sa
 	}
 }
 
+func createPodIdentityAssociation(ctx context.Context, f *framework.Framework, sa *v1.ServiceAccount, roleArn string) func(context.Context) error {
+	framework.Logf("Creating Pod Identity Association for ServiceAccount %s with role %s", sa.Name, roleArn)
```

Comment:
Could you check if we print role ARNs in other utils? Might want to avoid printing them to not leak sensitive information

Reply:
We do in this `waitUntilRoleIsAssumableWithWebIdentity` and other places too so it seems OK. If not, let me know and we can maybe change all of them as well